packages/smooth_app/lib/pages/onboarding/consent_analytics_page.dart

```diff
@@ -0,0 +1,188 @@
+
+class ConsentAnalytics extends StatelessWidget {
+  const ConsentAnalytics({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    final Size size = MediaQuery.of(context).size;
+    final LocalDatabase localDatabase = context.watch<LocalDatabase>();
+    final UserPreferences userPreferences = context.watch<UserPreferences>();
+    final AppLocalizations appLocalizations = AppLocalizations.of(context)!;
+    const Color shadowColor = Color.fromARGB(144, 0, 0, 0);
+    const Color bodyColor = Color.fromARGB(174, 19, 18, 18);
+    const String assetName = 'assets/onboarding/analytics.svg';
+    return Scaffold(
+      body: Column(
+        mainAxisAlignment: MainAxisAlignment.center,
+        children: <Widget>[
+          SizedBox(
+              height: size.height * 0.2,
+              width: size.width * 0.45,
+              child: SvgPicture.asset(
+                assetName,
+                semanticsLabel: 'Analytics Icons',
+                fit: BoxFit.contain,
+              )),
+          SizedBox(
+            height: size.height * 0.01,
+          ),
+          Align(
+            alignment: Alignment.center,
+            child: Text(
+              appLocalizations.consent_analytics_title,
+              style: Theme.of(context)
+                  .textTheme
+                  .headline2!
+                  .apply(color: Colors.black),
+            ),
+          ),
+          SizedBox(
+            height: size.height * 0.034,
+          ),
+          ConstrainedBox(
+            constraints: BoxConstraints(
+              maxWidth: size.width * 0.8,
+            ),
+            child: Text(
+              appLocalizations.consent_analytics_body1,
+              textAlign: TextAlign.center,
+              style: Theme.of(context)
+                  .textTheme
+                  .displaySmall!
+                  .apply(color: bodyColor),
+            ),
+          ),
+          SizedBox(
+            height: size.height * 0.03,
+          ),
+          ConstrainedBox(
+            constraints: BoxConstraints(
+              maxWidth: size.width * 0.8,
+            ),
+            child: Text(
+              appLocalizations.consent_analytics_body2,
+              textAlign: TextAlign.center,
+              style: Theme.of(context)
+                  .textTheme
+                  .displaySmall!
+                  .apply(color: bodyColor),
+            ),
+          ),
+          SizedBox(
+            height: size.height * 0.02,
+          ),
+          //Authorize Button
+          InkWell(
+            borderRadius: BorderRadius.circular(25.0),
+            onTap: () {
+              _analyticsLogic(true, userPreferences, localDatabase, context);
+            },
+            child: Ink(
+              height: size.height * 0.06,
+              width: size.width * 0.7,
+              decoration: BoxDecoration(
+                  color: Colors.green,
```

Comment:
We also have a special green color constant

Reply:
completed